### ISSUE TITLE
feature request/clarification: forwarding flows downstream

### ISSUE THREAD

**Issue by roycewilliams** [VerizonDigital/vflow#119]
If vflow does not support downstream forwarding - forwarding in specific flow formats, to multiple downstream flow-analysis systems - please consider this a feature request for that.

If vflow *does* support this, please consider this a request for clarification in the documentation.

**Comment by danielkim-z** [comment]
Hello,
vFlow already has a feature to replicate the IPFIX traffic to an external destination. Is this the feature you are looking for? Please note that mirroring only supports single destination at the moment.

Thank you.

**Comment by roycewilliams** [comment]
Yes, this is exactly the information I was looking for. Where is it located in the documentation? I want to understand why I could not find it.

**Comment by danielkim-z** [comment]
It appears that we do not have a separate documentation on the mirror feature. There are three configuration options related to mirroring; 

> ipfix-mirror-addr | IPFIX 3rd party collector address
> ipfix-mirror-port | IPFIX 3rd party collector port
> ipfix-mirror-workers | IPFIX replicator concurrent packet generator

Please let us know if you have any questions.
